src/game_simulation/game.py

```diff
@@ -0,0 +1,119 @@
+import random
+import numpy as np
+
+# TODO that is only game simulation - delete later, so it is not refactored. Please, don't read
+
+
+class Game:
+
+    def __init__(self):
+        self.map = np.zeros(shape=(5, 5), dtype='int32')
+        self.player_numbers = np.array([])
+        self.player_positions = [[-1, -1], [-1, -1], [-1, -1]]
+
+    def add_player(self, player_number, position=None):
+        if player_number in self.player_numbers:
+            raise ValueError("Player with this number already exists.")
+
+        if not position:
+            self.add_random_position(player_number)
+        else:
+            if self.map[position[0], position[1]] == 0:
+                raise ValueError("The position is not empty.")
+            else:
+                self.map[position[0], position[1]] = player_number + 1
+                self.player_positions[player_number] = position
+
+        self.player_numbers = np.append(self.player_numbers, player_number)
+
+    def add_random_position(self, player_number):
+        while True:
+            x = random.randint(0, 4)
+            y = random.randint(0, 4)
```

Comment:
Some magic numbers 😄 I think you can leave them. But to I wouldn't be angry if you did that